feature/experimental/gribi/ate_tests/backup_nhg_action/backup_nhg_action_test.go

```diff
@@ -412,7 +412,9 @@ func testDecapEncap(ctx context.Context, t *testing.T, args *testArgs) {
 	}
 	t.Run("ValidateDecapPath", func(t *testing.T) {
 		t.Log("Validate traffic after decap is recieved on port4 and no traffic on other flows/ate ports")
-		validateTrafficFlows(t, args.ate, []*ondatra.Flow{decapFLow}, []*ondatra.Flow{baseFlow, encapFLow}, decapFlowFliter)
+		if !deviations.SecondaryBackupPathTrafficFailover(ondatra.DUT(t, "dut")) {
+			validateTrafficFlows(t, args.ate, []*ondatra.Flow{decapFLow}, []*ondatra.Flow{baseFlow, encapFLow}, decapFlowFliter)
```

Comment:
Reuse args.dut here instead of making a call to ondatra.

Reply:
Updated with args.dut.